elements/storytelling/src/helpers/render-html-string.js

```diff
@@ -138,3 +138,34 @@ export function convertAttributeValueBasedOnItsType(
 
   return convertedValue;
 }
+
+/**
+ * Parse Nav and generate a new Element Node
+ *
+ * @param {Array<Element>} html - List of html elements
+ * @param {Array} nav - List of nav elements
+ * @returns {Element[]} An array of processed DOM nodes after adding navigation.
+ */
+export function parseNav(html, nav) {
+  const parser = new DOMParser();
+  let navIndex = 0;
+
+  const navHtml = `
+    <div class="navigation">
+      <div class="container">
+        <ul>
+          ${nav.map(
+            ({ id, title }) =>
+              `<li class="nav-${id}"><a href="#${id}">${title}</a></li>`
+          )}
```

Comment:
Please remove the automatically generated comma (`,`), by either an empty string or a differebt character, e.g.
```
          ${nav
            .map(
              ({ id, title }) =>
                `<li class="nav-${id}"><a href="#${id}">${title}</a></li>`
            )
            .join("")}
        </ul>
```